### ISSUE TITLE
"Unexpected token import" when running babel-watch

### ISSUE THREAD

**Issue by stoikerty** [kmagiera/babel-watch#2]
Hey @kmagiera 

Thanks for creating this package. It seems very useful!
Unfortunately I wasn't able to make it work so far. I have used the following command:

```
babel-watch --presets es2015 --watch src webpack/production.js
```

as well as other variations like these:

```
babel-watch --presets "es2015" --watch src webpack/production.js
babel-watch --presets=es2015 --watch src webpack/production.js
babel-watch --presets="es2015" --watch src webpack/production.js
```

I am getting the following error message:

``` js
C:\repos\my_repo\webpack\production.js:1
(function (exports, require, module, __filename, __dirname) { import webpack from 'webpack';
                                                              ^^^^^^
SyntaxError: Unexpected token import
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Module._extensions..js (module.js:432:10)
    at loader (C:\repos\my_repo\node_modules\babel-watch\runner.js:14:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\repos\my_repo\node_modules\babel-watch\runner.js:22:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:457:10)
    at process.<anonymous> (C:\repos\my_repo\node_modules\babel-watch\runner.js:38:21)
    at emitTwo (events.js:87:13)
```

The file in question starts like this:

``` js
import webpack from 'webpack';
```

I also have a `.babelrc`-file but it doesn't seem to pick it up.

Any idea how I can get it to work?
I'm currently using nodemon with the following command (and yes, the performance is pretty bad):

```
nodemon --exec babel-node --debug webpack/production.js --watch webpack/production.js
```

**Comment by kmagiera** [comment]
Not in front of my computer at the moment but from your issue description it looks like watcher is not set for the file you're getting this error. For babel-watch it's important that all the files that needs to be transpiled with babel are also watched. From the command you posted it looks like you only watch files under src and if webpack directory contains files that need transpiration (like webpack/production.js) you need to make sure that the file is added to the watcher path (e.g by adding `--watch webpack` to your command)

I hope this advice will unblock you. Ive been thinking now for a while about a solution that would get rid of the limitation I've mentioned above so that you can only provide an entry point is file for babel-watch and the tool will figure out deps on its own

**Comment by kmagiera** [comment]
@stoikerty I hope your issue is now resolved as I haven't heard back from you. So I will go ahead and close this github issue - feel free to reopen if you have further questions

**Comment by stoikerty** [comment]
I have just come back to the issue and tried your solution. I'm not sure how to add 2 folders to the watcher though as I'm not getting it to work.

When I use `babel-watch --watch src --watch webpack toolkit/webpack/development.js`, it seems to just get stuck as nothing happens at all.
If I use `babel-watch --watch src webpack toolkit/webpack/development.js`, the watcher thinks that `webpack` is a module and gives me the error `Cannot find module`.

I'm trying to implement this in my **[universal-dev-toolkit](https://github.com/stoikerty/universal-dev-toolkit)** and created [a PR that tries to implement babel-watch](https://github.com/stoikerty/universal-dev-toolkit/pull/3). Maybe you could have a look? Thanks!

**Comment by kmagiera** [comment]
I think you actually want to watch `toolkit/webpack` not just `webpack`

To watch multiple paths you should place `--watch` in tront of each one. So the command you want to use should be:

```
babel-watch --watch src --watch toolkit/webpack toolkit/webpack/development.js
```

**Comment by stoikerty** [comment]
Thanks! indeed I was watching the wrong folder!

The command is running now, but I still get an error when running `npm run dev`. It's almost as if it's not picking up on the babel-presets from `.babelrc`. I've also tried with `--presets es2015` but I get the same error.

``` js
M:\_repos\universal-dev-toolkit\src\client\views\AppBody\Component.js:1
(function (exports, require, module, __filename, __dirname) { import React from 'react';
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Module._extensions..js (module.js:432:10)
    at loader (M:\_repos\universal-dev-toolkit\node_modules\babel-watch\runner.js:14:5)
    at Object.require.extensions.(anonymous function) [as .js] (M:\_repos\universal-dev-toolkit\node_modules\babel-watch\runner.js:22:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (AppBody.js:5:1)
```

_edit: I moved the discussion to a [new issue](https://github.com/kmagiera/babel-watch/issues/6)_

**Comment by harm-meijer** [comment]
What am I doing wrong here?

Have `"babel-watch": "^7.0.0"` in devDependencies and `"dev": "babel-watch --presets es2015 ./index.js",` in scripts of package.json, have an index.js with `import express from "express";` but running `yarn dev` gives me:

```
import express from "express";
       ^^^^^^^

SyntaxError: Unexpected identifier
```

**Comment by Adam-Edry** [comment]
I ran into the same issue as @harm-meijer and it turned out to be caused by having a space in my path. Changing the directory let me run things as usual
